src/css/styles.css

```diff
@@ -1,3 +1,106 @@
+#title {
+    text-align: center;
+    font-family: "Palatino Linotype", "Book Antiqua", Palatino, serif;
```

Comment:
use Lato only if you don't need to import a font type

Reply:
done